cookiecutter/hooks.py

```diff
@@ -84,6 +96,13 @@ def run_script_with_context(script_path, cwd, context):
     :param cwd: The directory to run the script from.
     :param context: Cookiecutter project template context.
     """
+    if '_no_hookcopy' in context and re.match(
```

Comment:
Ah ok, I take ☝️ back, it is useful to have access to the template directory in the hooks.
I think the docs would benefit from an example that shows this off.